scripts/make-guac.sh

```diff
@@ -575,78 +635,43 @@ then
     mkdir /var/tmp/guacamole
 fi
 
-# Start services
-log "Attempting to start proxy-related services"
-for SVC in rsyslog guacd tomcat8
-do
-    log "Stopping and starting ${SVC}"
-    /sbin/service ${SVC} stop && /sbin/service ${SVC} start
-    if [[ $? -ne 0 ]]
-    then
-      die "Failed to start ${SVC}."
-    fi
-done
-
 
 #Add custom URLs to Guacamole login page using Guac extensions.
 if ( [[ -n "${URL_1}" ]] || [[ -n "${URL_2}" ]] )
 then
-    log "Attempting to add HTML links from parameter input to Guacamole login page"
-    log "Writing Guac manifest file"
-    (
-        printf "{\n"
-        printf "\"guacamoleVersion\" : \"$GUAC_VERSION\",\n"
-        printf "\"name\" : \"Custom Extension\",\n"
-        printf "\"namespace\" : \"custom-extension\",\n"
-        printf "\"html\" : [ \"custom-urls.html\" ]\n"
-        printf "}\n"
-    ) > /etc/guacamole/extensions/guac-manifest.json
-    log "Writing Guac html extension file to add in custom URLs"
-    (
-        printf "<meta name=\"after\" content=\".login-ui .login-dialog\">\n"
-        printf "\n"
-        printf "<div class=\"welcome\">\n"
-        printf "<p>\n"
-        printf "<a target=\"_blank\" href=\"$URL_1\">$URLTEXT_1</a>\n"
-        printf "</p>\n"
-        printf "<p>\n"
-        printf "<a target=\"_blank\" href=\"$URL_2\">$URLTEXT_2</a>\n"
-        printf "</p>\n"
-        printf "</div>\n"
-    ) > /etc/guacamole/extensions/custom-urls.html
-
-   #Zip extension files into custom.jar
-   cd "/etc/guacamole/extensions"
-   zip "custom.jar" "custom-urls.html" "guac-manifest.json"
-   log "Successfully added URL(s) to Guacamole login page"
+    write_manifest
+    write_links
 else
+
     log "URL parameters were blank, not adding links"
 fi
 
-#Customize branding to title page. Attempt to use extensions, but does not seem to take. Search and replace file en.json instead. Not preferred, but will do for now.
+
+#Add custom branding text to title page.
 if [[ -n "${BRANDTEXT}" ]]
 then
-   sleep 15
-   sed -i "s|Apache Guacamole|$BRANDTEXT|g" /var/lib/tomcat8/webapps/ROOT/translations/en.json
-        if [[ $? -ne 0 ]]
-        then
-            log "sed statement failed to set custom branding text: ${BRANDTEXT}"
-        fi
-   log "Successfully added branding text to Guacamole login page"
+    log "Writing Guac translations extension file to add in custom branding text"
+    mkdir -p /etc/guacamole/extensions/translations
+    if [ ! -f "/etc/guacamole/extensions/guac-manifest.json" ]
+    then
+        write_manifest
+        write_brand
```

Comment:
How about just calling `write_manifest` here, deleting the `else` block, and moving `write_brand` outside the `if` block? i.e.:

```
    if [ ! -f "/etc/guacamole/extensions/guac-manifest.json" ]
    then
       write_manifest
    fi
    write_brand
```